Public/Sdk/Public/Managed/Frameworks/netcoreapp3.0/netcoreapp3.0.dsc

```diff
@@ -65,6 +65,12 @@ export const framework : Shared.Framework = {
     // Deployment style for .NET Core applications currently defaults to self-contained
     applicationDeploymentStyle: "selfContained",
     runtimeContentProvider: runtimeContentProvider,
+    
+    conditionalCompileDefines: [
+        "NET_CORE",
+        "NET_COREAPP",
+        "NET_COREAPP_30"

```

Comment:
>NET_COREAPP_30 [](start = 9, length = 14)

Will this conflict with my approach in #662?

Reply:
hehe, can only add messages after creating PR. I know, see below. I'll wait for yours and merge. Just wanted to get this out so I have a change for Kristijan to review.